common/models/environment_link_request.go

```diff
@@ -0,0 +1,38 @@
+package models
+
+import (
+	"fmt"
+
+	swagger "github.com/zpatrick/go-plugin-swagger"
+)
+
+type CreateEnvironmentLinkRequest EnvironmentLinkRequest
+
+type DeleteEnvironmentLinkRequest EnvironmentLinkRequest
```

Comment:
[nit] we probably don't even need these extra types, seems reasonable to just use the `EnvironmentLinkRequest` in both create/delete scenarios